.travis.yml

```diff
@@ -51,21 +50,18 @@ addons:
     notification_email: sledgehammer999@qbittorrent.org
   apt:
     sources:
-      # sources list: https://github.com/travis-ci/apt-source-whitelist/blob/master/ubuntu.json
+      # sources list: https://github.com/travis-ci/apt-source-safelist/blob/master/ubuntu.json
       - ubuntu-toolchain-r-test
-      #- boost-latest
       - sourceline: 'ppa:qbittorrent-team/qbittorrent-stable'
-      - sourceline: 'ppa:beineri/opt-qt551-trusty'
-      - sourceline: 'ppa:adrozdoff/cmake'
     packages:
-      # packages list: https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise
+      # packages list: https://github.com/travis-ci/apt-package-safelist/blob/master/ubuntu-trusty
+      - [gcc-5, g++-5]
```

Comment:
Xenial has gcc 5 by default: https://packages.ubuntu.com/search?suite=xenial&arch=any&searchon=names&keywords=g%2B%2B

Reply:
the default installation is gcc 5.4.0, while here updates it to 5.5.0. I'll use the default one.